src/main/java/com/google/graphgeckos/dashboard/datatypes/BuildBotData.java

```diff
@@ -95,4 +111,49 @@ public Timestamp getTimestamp() {
     return timestamp;
   }
 
+  /**
+   * Sets the name of the builder bot.
+   *
+   * @param name the new value for the {@code name} field. Should not be null.
+   */
+  public void setName(@NonNull String name) {
+    this.name = name;
+  }
+
+  /**
+   * Sets the list of logs for each compilation stage.
+   *
+   * @param logs the new value for the {@code logs} field. Should not be null.
+   */
+  public void setLogs(@NonNull List<Log> logs) {
+    this.logs = new ArrayList<>(logs);
+  }
+
+  /**
+   * Sets the compilation status of the builder.
+   *
+   * @param status the new value for the {@code status} field. Should not be null.
+   */
+  public void setStatus(@NonNull BuilderStatus status) {
+    this.status = status;
+  }
+
+  /**
+   * Sets the commit hash of the commit tested by the current buildbot.
+   *
+   * @param commitHash the new value for the {@code commitHash} field. Should not be null.
+   */
+  public void setCommitHash(@NonNull String commitHash) {
+    this.commitHash = commitHash;
+  }
+
+  /**
+   * Sets the timestamp of the build.
```

Comment:
Nit: remove this part of the comment because names of setters are self-explanatory

Reply:
Done. Thank you!